app/views/layouts/application.html.erb

```diff
@@ -5,10 +5,12 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag    'application', media: 'all' %>
```

Comment:
good catch making your css load correctly by moving it to the end of the list.

Reply:
That took more than a second to figure out, but, it was a mega relief when I did.